nilearn/decoding/decoder.py

```diff
@@ -824,6 +824,7 @@ def decision_function(self, X):
         """
         # for backwards compatibility - apply masker transform if X is
         # niimg-like or a list of strings
+        check_is_fitted(self)
```

Comment:
```suggestion
        check_is_fitted(self)
        # for backwards compatibility - apply masker transform if X is
        # niimg-like or a list of strings
```

Just moved this above the comment